### PR TITLE
feat(styles): Update password reset templates

### PR DESCRIPTION
**Pull request by vbudhram**
This PR updates password flow email templates. 

Reference https://github.com/mozilla/fxa-content-server/issues/3467
Reference https://github.com/mozilla/fxa-content-server/pull/3559

### CODE REVIEW

**Comment by vbudhram** [comment]
@vladikoff @shane-tomlinson r? Thanks!

File: config.js [hunk 1]
```diff
@@ -129,6 +129,11 @@ var conf = convict({
       format: String,
       default: 'https://www.mozilla.org/firefox/ios/'
     },
+    signInUrl: {
```
Comment (vladikoff) [inline config.js]: we will need to add this to https://github.com/mozilla/fxa-auth-server/blob/master/config/index.js#L367 as well

**Comment by vladikoff** [comment]
needs email on acid check...

**Comment by vbudhram** [comment]
Hey @vladikoff, here are results from email on acid tests. Went through several screens and it looked good. Detailed results are on the site if you want to take a peep. Let me know what you think, thanks!

* Reset your Firefox Account password
screen shot 2016-03-10 at 10 20 44 pm
screen shot 2016-03-10 at 10 21 08 pm
screen shot 2016-03-10 at 10 20 35 pm

* Your Firefox Account password has been reset
screen shot 2016-03-10 at 10 31 35 pm
screen shot 2016-03-10 at 10 25 58 pm
screen shot 2016-03-10 at 10 26 22 pm

**Comment by vladikoff** [comment]
I checked other email on acid reports, looks good!

**Pull request merged**